test/hemi.tokenlist.test.js

```diff
@@ -121,12 +121,13 @@ describe("List of tokens", function () {
         const client = clients[chainId];
         const tokenAddress =
           extensions?.bridgeInfo?.[client.chain.sourceId].tokenAddress;
-        if (!tokenAddress) {
+        const remoteToken = await getRemoteToken(client, address).catch(
+          () => null,
+        );
+        if (!remoteToken) {
```

Comment:
If `tokenAddress` is defined but `remoteToken` is not, should this fail? As is, it will be skipped silently

Reply:
Fixed!